packages/trackerless-network/src/logic/StreamrNode.ts

```diff
@@ -95,6 +95,8 @@ export interface StreamrNodeConfig {
     acceptProxyConnections?: boolean
 }
 
+const NETWORK_SPLIT_AVOIDANCE_LIMIT = 4
```

Comment:
This constant could be in `StreamEntryPointDiscovery` and we don't need `this.config.networkSplitAvoidanceLimit`